bibtex-actions.el

```diff
@@ -417,10 +421,67 @@ TEMPLATE."
 ;; something equivalent.
 (defun bibtex-actions-get-key-org-cite ()
   "Return key at point for org-cite citation-reference."
-  (when (eq major-mode 'org-mode)
-    (let ((elt (org-element-context)))
-      (if (eq (car elt) 'citation-reference)
-          (org-element-property :key elt)))))
+  (when-let (((eq major-mode 'org-mode))
+             (elt (org-element-context)))
+    (pcase (org-element-type elt)
+      ('citation-reference
+       (org-element-property :key elt))
+      ('citation
+       (org-cite-get-references elt t)))))
+
+;;; Embark
+
+(defun bibtex-actions-citation-key-at-point ()
+  "Return citation keys at point as string for `embark'."
+  (when-let ((key (or (bibtex-actions-get-key-org-cite)
+                      (bibtex-completion-key-at-point))))
+    (cons 'citation-key (if (listp key) (string-join key ";") key))))
+
+(with-eval-after-load 'embark
+  (add-to-list 'embark-target-finders 'bibtex-actions-citation-key-at-point)
```

Comment:
Another question (and it is a question; I'm not sure) for both @publicimageltd and @ilupin:

Right now, one effectively cannot get to the default action directly. One must open `embark`, and then hit `RET`.

And both `bibtex-actions-at-point` and `embark-act` do the exact same thing; call `embark-act`.

Is that the right UX?

Reply:
PS - see this new comment, and linked commit, from @minad; might be relevant?

https://github.com/hlissner/doom-emacs/pull/4664#pullrequestreview-682442636

Reply:
I'm fine with adding a defcustom to make `embark-dwim` the default. It will be consistent with the case where `embark` is not installed.